ios/FluentUI/Popup Menu/PopupMenuItem.swift

```diff
@@ -99,6 +100,7 @@ open class PopupMenuItem: NSObject, PopupMenuTemplateItem {
         self.executionMode = executionMode
         self.onSelected = onSelected
         self.isAccessoryCheckmarkVisible = isAccessoryCheckmarkVisible
+        self.fluentTheme = FluentTheme.shared
```

Comment:
we shouldn't do this right? this may make the popupmenuitem have the wrong color?

Reply:
That'll be addressed in a later PR to freshen up the stale popup menu.